core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -103,18 +104,20 @@ public Workflow getWorkflowById(String workflowId, boolean includeTasks) {
      */
     public List<Workflow> getWorkflowsByCorrelationId(String correlationId, boolean includeTasks) {
```

Comment:
This method is called from the API, so using parallel stream processing here to improve performance.